src/Fulfilment/Model/Order.php

```diff
@@ -114,7 +114,7 @@ public static function fromPdkOrder(?PdkOrder $pdkOrder): self
                         );
                     })
                     ->all(),
-                'orderNotes'                  => $pdkOrder->orderNotes,
+                'notes'                       => $pdkOrder->notes,
```

Comment:
Dit is de fix zelf, `orderNotes` bestond niet op PdkOrder. Om van de verwarring af te komen heb ik meteen de property overal rechtgetrokken (met backwards compatibility via `$deprecated`)